src/ui/groups/detail/GroupDetailTabContents.js

```diff
@@ -25,14 +26,16 @@ class GroupDetailTabContents extends React.Component {
   }
 
   renderRows() {
-    return this.props.contentReferences.map(item => (
-      <tr key={item.code}>
-        <td>{item.name}</td>
-        <td>{item.code}</td>
-        <td>{item.type}</td>
-        <td>{item.lastEdit}</td>
-      </tr>
-    ));
+    const DATE_FORMAT = 'DD/MM/YYYY';
```

Comment:
lets not declare constants that are only used once.

Reply:
i've fixed the component form your suggestion @nicpuddu